contain_learning/Dockerfile

```
@@ -0,0 +1,5 @@
+FROM centos:latest
+RUN yum install python36 -y
+RUN pip3 install numpy
+RUN pip3 install keras
+RUN pip3 install sklearn 
```

Identified a potential vulnerability, here’s a suggested change for The 'pip3 install' commands do not specify versions for the packages, which can lead to non-deterministic builds.:

```suggestion
RUN pip3 install numpy==1.21.0 keras==2.4.3 sklearn==0.24.2
```